irohad/ordering/impl/ordering_service_transport_grpc.cpp

```diff
@@ -33,14 +39,16 @@ grpc::Status OrderingServiceTransportGrpc::onTransaction(
   if (subscriber_.expired()) {
     log_->error("No subscriber");
   } else {
-    subscriber_.lock()->onTransaction(*factory_.deserialize(*request));
+    subscriber_.lock()->onTransaction(
+        shared_model::proto::Transaction(*request));
   }
 
   return ::grpc::Status::OK;
 }
 
 void OrderingServiceTransportGrpc::publishProposal(
-    model::Proposal &&proposal, const std::vector<std::string> &peers) {
+    shared_model::proto::Proposal &&proposal,
```

Comment:
You can use shared_model namespace at the top of the file

Reply:
Doesn't change much